lusidtools/lpt/create_orders.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+
+from lusidtools.lpt import lpt
+from lusidtools.lpt import lse
+from lusidtools.lpt import stdargs
+
+NAME = "name"
+TOOLNAME = "orders_create"
+TOOLTIP = "Create Orders"
+
+
+def parse(extend=None, args=None):
+    return (
+        stdargs.Parser("Upsert Orders", ["filename", "limit", "test"])
+        .add("input", nargs="+")
+        .add(
+            "--mappings",
+            nargs="+",
+            help="column name mappings of the form TICKER=col1 etc",
+        )
+        .add(
+            "--identifiers",
+            nargs="+",
+            default=["ClientInternal", "Figi"],
+            help="Identifier types provided",
+        )
+        .extend(extend)
+        .parse(args)
+    )
+
+
+def process_args(api, args):
+    aliases = {
+        "CINT": "ClientInternal",
+        "FIGI": "Figi",
+        "RIC": "P:Instrument/default/RIC",
+        "TICKER": "P:Instrument/default/Ticker",
+        "ISIN": "P:Instrument/default/Isin",
+    }
+
+    if args.input:
+        df = pd.concat(
+            [lpt.read_input(input_file, dtype=str) for input_file in args.input],
+            ignore_index=True,
+            sort=False,
+        )
+
+        if args.mappings:
+            df.rename(
+                columns=dict(
+                    [
+                        (s[1], aliases.get(s[0], s[0]))
+                        for s in [m.split("=") for m in args.mappings]
+                    ]
+                ),
+                inplace=True,
+            )
+
+        prop_keys = [col for col in df.columns.values if col.startswith("P:")]
+
+        identifiers = [col for col in df.columns.values if col in args.identifiers]
+
+        # Identifiers have to be unique
+        df = df.drop_duplicates(identifiers)
```

Comment:
These lines are an exact copy of lines 18-67 in create_instr.py. Is it possible to move both of these functions into a common module that can be imported by both create_instr.py and create_orders.py?

Reply:
Have factored out most into a new method in lpt.py